### DISCUSSION TITLE
Exigencias de complejidad

### DISCUSSION THREAD

**Discussion by sofiacorreabusquets** [IIC2133-PUC/2024-1#137] [Tarea 2] [answered]
Hola ☺️ 

Tengo una pregunta sobre la sección _Evaluación_ de la T2, donde dice "en esta tarea cada sección indica algún tipo de complejidad algorítmica la cual deberas respetar." No encontré esa indicación en ninguna de las partes del enunciado... Veo que en https://github.com/orgs/IIC2133-PUC/discussions/136#discussioncomment-9462779 indican que la solución debe ser lineal respecto del espacio de búsqueda para las partes 1 y 3, pero, ¿cuál es la complejidad permitida para la parte 2? Muchas gracias de antemano 🙌

**Comment by sebacarrascop** [comment, marked as answer]
hola!, se espera que utilicen backtracking que puede llegar a tener una complejidad exponencial. Recuerda que en caso de necesitarlos, hay métodos que permiten reducir la cantidad de combinaciones exploradas y, por lo tanto, mejorar la eficiencia en la práctica.

**Reply by sofiacorreabusquets** [reply]
súper, gracias
